server/src/project_service.ts

```diff
@@ -172,4 +171,33 @@ export class ProjectService {
       });
     }
   }
+
+  /**
+   * Determine valid directories where `@angular/language-service` could be
+   * found.
+   * @param probeLocation
+   */
+  private getPluginProbeLocations(probeLocation?: string): string[] {
+    const versionProvider = new NgVersionProvider(probeLocation);
+    const pluginProbeLocations = [];
+    const localVersion = versionProvider.localVersion;
+    if (localVersion) {
```

Comment:
Does the extension work with 8.x.x or bellow version? Consider to ignore unsupported versions. This will prevent the issues like #400.

Reply:
yes, I agree with that. Will do so in a follow up PR.